fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/WeightCheckRequestHandler.java

```diff
@@ -71,6 +73,9 @@ public boolean handleRequest(
             return next.handleRequest(context, request, druidQuery, response);
         }
 
+        ((BardQueryInfo) RequestLog.retrieve(BardQueryInfo.class)).incrementCountFor(
+                BardQueryInfo.WEIGHT_CHECK
+        );
```

Comment:
You could invert this and add a helper method on BardQueryInfo to do the increments.